reviewer_readme.md

```diff
@@ -1,46 +1,116 @@
+![open in github](https://drive.google.com/uc?id=0BwL_X3dS1kN_a2FUYUFKdHNZSGc)
 
-### Denying Content
 
-To deny a pull request, click the "Close Pull Request" Button. The author will be notified via email, but we encourage you to also send a message as to why their contribution is not going to be merged into the site.
+--------
 
-### Merging Content
+### Step 3: Review tests
 
-Discuss the content with the author in the pull request. When you both agree that the content is ready, and Travis CI jobs are green, it is safe to merge.
+![failed tests](https://drive.google.com/uc?id=0BwL_X3dS1kN_NmpXbDFZMGdIOTg)
 
-### View on Integration site
+The author's submission will automatically be tested by a product called Travis CI that will verify if the submission is valid. Submissions that fail these tests should be rejected (aka. closed). Just because a submission passes these tests does not mean you absolutely should accept (merge) it. Click on "Details" for more information. We recommend that automated tests be used only as a guideline and not as a definitive solution. Automated test suites can always be better, and reviewers should look at the results of http://int-androidhub.intel.com before releasing to production.
+
+--------
+
+### Step 4: Accept or Deny
+
+##### Accept
+
+![ready to merge](https://drive.google.com/uc?id=0BwL_X3dS1kN_eFVlVktXVXlhUG8)
+
+If a submission's content looks good and the Travis CI tests are green, it is safe to merge. Click the button on the Pull Request page labeled "Merge pull request". This will merge the content into the `master` branch and tell Travis CI to automatically deploy the site to the private Integration environment.
+
+##### Deny
+![pr comment](https://drive.google.com/uc?id=0BwL_X3dS1kN_cGJiaWRwb05Dbzg)
+
+To deny a submission, click the "Close Pull Request" Button. The author will be notified via email. We encourage you to also send a message as to why the submission is not going to be merged into the site at this time.
+
+--------
+
+### Step 5: Auto-Deploy to Integration
+
+![integration site](https://drive.google.com/uc?id=0BwL_X3dS1kN_a2I1VU5tUHdMc2c)
 
 https://int-androidhub.intel.com
 
-The above url is the integration site. The username and password for it were given to you by project staff directly. This site is automatically deployed upon a successful merge to the master branch on the github repository.
+The above url is the integration site. A successful merge to the master branch on github will automatically deploy the submission to the Integration environment. The username and password for it were given to you by project staff.
+
+--------
+
+### Step 6: Release to Production
```

Comment:
I think it’s worth explaining what “production deployments” mean. Something like “once this is complete the changes/posts will be visible the the general public on androidhub.intel.com.